.travis.xacro_test.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
-for x in $(find "$TARGET_REPO_PATH" -name '*.xacro'); do
+status=0
+[[ "$ROS_DISTRO" < "melodic" ]] && xacro_args='--inorder'
```

Comment:
sh might work with
```suggestion
[ "$ROS_DISTRO" \< "melodic" ] && xacro_args='--inorder'
```
(not POSIX-compliant)

Reply:
I can't remember why I used `sh` over `bash`....but with you hint it could stay like this = no changes needed in `industrial_ci`....thus, I'm taking this solution...thanks